Makefile

```diff
@@ -94,6 +94,7 @@ images:
 	bazel run \
 		--stamp \
 		--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
+		--@io_bazel_rules_go//go/config:pure \
 		//build:server-images
 
```

Comment:
Thanks @wallrj ! The cgo & dynamic image are both for FIPS- since that is a closed source project, I have not added any information about that here. Perhaps the parameterization would also help someone who is building `cert-manager` for themselves and also need to be able to specify a different image or use `cgo`. The implementation detail for FIPS is in our private Notion that I think I can link here https://www.notion.so/jetstack/cert-manager-FIPS-compliant-build-e44ad707abb448ebada14d235a95cf4d as you'd need to log in anyway.